lib/task-data/base-tasks/diag-firmware-update.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017, Dell EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Update firmware via diag',
+    injectableName: 'Task.Base.Update.Firmware',
+    runJob: 'Job.Diag.Update.Firmware',
```

Comment:
if it's related to diag, and particularly,  or if it's specific to DellEMC, or EMC GHE, or EMC IVE Diag, it could be reflected in the job name